orchestrator/cli/generator/generator/migration.py

```diff
@@ -134,25 +140,39 @@ def generate_product_migration(context: dict) -> None:
     if "data" not in heads:
         create_data_head(context=context, depends_on=heads["schema"])
 
-    if migration_file := create_migration_file(message=f"add {config['name']}", head="data"):
-        if revision_info := get_revision_info(migration_file):
-            if "fixed_inputs" in config and config["fixed_inputs"]:
-                fixed_input_values = [
-                    [(fixed_input["name"], str(value)) for value in fixed_input["values"]]
-                    for fixed_input in config["fixed_inputs"]
-                    if "values" in fixed_input
-                ]
-                fixed_input_combinations = list(itertools.product(*fixed_input_values))
-                product_variants = [
-                    (" ".join([config["name"]] + [value for name, value in combination]), combination)
-                    for combination in fixed_input_combinations
-                ]
-            else:
-                product_variants = [((config["name"]), ())]
-            template = environment.get_template("new_product_migration.j2")
-            content = template.render(product=config, product_variants=product_variants, **revision_info)
-            writer(migration_file, content)
-            update_subscription_model_registry(environment, config, product_variants, writer)
-
-    else:
+    if not (migration_file := create_migration_file(message=f"add {config['name']}", head="data")):
         logger.error("Could not create migration file")
+        return
+
+    if not (revision_info := get_revision_info(migration_file)):
+        logger.error("Could not get revision info from migration file", migration_file=migration_file)
+        return
+
+    if "fixed_inputs" in config and config["fixed_inputs"]:
+        fixed_input_values = [
+            [(fixed_input["name"], str(value)) for value in fixed_input["values"]]
+            for fixed_input in config["fixed_inputs"]
```

Comment:
to batch with above comment
```suggestion
            for fixed_input in fixed_inputs
```